includes/ets_user_qa_function.php

```diff
@@ -197,18 +198,21 @@ public function ets_ask_qustion_tab() {
 					__( 'Please <a href="%s">login</a> to post questions', 'product-questions-answers-for-woocommerce' ),
 					apply_filters( 'wc_add_qa_login_url', wp_login_url(home_url( $wp->request )) )
 				);
-			?>
-			<?php  
+			 
 		} 
 		
-		$this->display_qa_listing();
+			?>
+		<div id="qa-tab-qa-listing">
+			<?php $this->display_qa_listing($container_id = '2'); ?>
```

Comment:
Why we are passing a value to the function like this `display_qa_listing($container_id = '2')`  you can pass `2` directly.

Reply:
Because I use this value dynamically in other place,so i use in variable .

Reply:
Wrong style! Don't assign variable while calling the function.

Either do this:
```
$container_id = '2';
display_qa_listing($container_id);
```
or this:
`display_qa_listing(2);`